src/App.js

```diff
@@ -100,7 +125,10 @@ function App() {
 
   return (
     <Router>
-      <div className="App container">
+      <div
+        className="App container"
+        aria-hidden={showModal} // if showModal is true, hide the rest of the app
```

Comment:
While this does work to hide the element and as such keeps the screen reader from hitting it, I would caution against this approach as the component still renders and any logic within it will run.  

As an example, I added a `console.log` within the Modal component and as you can see below that fired and logged to the console with the hidden attribute vs conditionally rendering the component using `showModal && <Modal />`

| aria-hidden | conditional render |
| ------------- | ------------- |
| <video src="https://user-images.githubusercontent.com/19564687/130313275-c0b568cd-f23d-4640-930f-3d225766c428.mov" />  | <video src="https://user-images.githubusercontent.com/19564687/130313301-ded67317-8e5a-4a24-81c4-19f219d3d017.mov" />  |

Reply:
This is an interesting approach. I'm not sure what the right way to approach this as well. Will have to research it.

Reply:
@abowler2 That is true! Hmm. For modals, drop down menus, etc. I've always done them this same way, having them render (or just be included in markup if vanilla JS) but shown/hidden with a CSS class, so later I could use CSS transitions to animate them fading or sliding in and out of view. The bootcamp I attended taught this method as well. Is this basically bad for performance in React? And is there a good alternate way you can still have an animation with the `showModal && <Modal />` approach?

Reply:
@yenly I think it comes from [this w3.org page](https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html). From what we could determine, the aria-modal attribute on the modal alone *should* take care of hiding things that are not the modal, but it is new so not fully supported everywhere. For full support, it's suggested to use aria-hidden="true" on "each element containing a portion of the inert layer" and that the modal must not be inside an aria-hidden layer. I found this general pattern pretty common: 
```
<div aria-hidden="true">the whole rest of the page</div>
<div aria-modal="true">the active modal</div>
```
Though what we weren't certain of is what percentage of screen reader users might have legacy software that doesn't fully support aria-modal, therefore how necessary aria-hidden really is!

Reply:
Thank you all for the helpful input, especially April with the gifs (very helpful and clear what the issue is.) Is this something we can discuss Sunday if there is time?